scenes/controller/unlock_controller/unlock_controller.gd

```diff
@@ -19,6 +19,16 @@ func _ready() -> void:
 ##############
 
 
+func _handle_on_substance_updated(observed_id: String, total_amount: int) -> void:
+	if total_amount >= 1:
+		var substance_data: SubstanceData = Resources.substance_datas.get(observed_id)
```

Comment:
Merge this only after this: https://github.com/TinyTakinTeller/GodotProjectZero/pull/25